src/node_main.cc

```diff
@@ -7,7 +7,7 @@ int wmain(int argc, wchar_t *wargv[]) {
   if (!IsWindows7OrGreater()) {
     fprintf(stderr, "This application is only supported on Windows 7, "
                     "Windows Server 2008 R2, or higher.");
-    exit(1);
+    exit(0xd8);     // code: ERROR_EXE_MACHINE_TYPE_MISMATCH
```

Comment:
I think you can also `#include <windows.h>` above and then use the macro directly, if you prefer.